.circleci/config.yml

```diff
@@ -62,8 +62,6 @@ jobs:
             cd build
             cmake ../ \
               -DCMAKE_VERBOSE_MAKEFILE=ON \
-              -DUSE_IBVERBS=ON \
-              -DUSE_LIBUV=ON \
```

Comment:
Why moving this to individual builds and distinguish `*` with `*-all-transports`?

Reply:
To ensure that the project can be built with and without any transport(that all preprocessor if-branches are valid)